core/src/main/scala/filodb.core/memstore/OnDemandPagingShard.scala

```diff
@@ -223,8 +223,12 @@ TimeSeriesShard(ref, schemas, storeConfig, shardNum, bufferMemoryManager, rawSto
         case req: TimeRangeChunkScan      =>  if (partition.numChunks > 0) {
                                                 val memStartTime = partition.earliestTime
                                                 if (req.startTime < memStartTime && partStartTime < memStartTime) {
+                                                  val toODP = TimeRangeChunkScan(req.startTime, memStartTime - 1)
+                                                  logger.debug(s"Decided to ODP time range $toODP for " +
+                                                    s"partID=${partition.partID} memStartTime=$memStartTime " +
```

Comment:
this is the data which is used to query Cassandra. ?

Reply:
Correct - this is.